bin/setup-wp.js

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env node
+const prompt = require('prompt-sync')();
+const {spawn} = require('child_process');
+const {execSync} = require('child_process');
+const files = require('./files.js');
+
+// Helpers
+const fgRed = '\x1b[31m';
+const fgGreen = '\x1b[32m';
+const fgBlue = '\x1b[34m';
+const fgMagenta = '\x1b[35m';
+const fgCyan = '\x1b[36m';
+
+// Functions
+const consoleOutput = (color, text) => {
+  console.log(color, text);
+};
+
+// ---------------------------
+//  1. Info prompting
+// ---------------------------
+
+let wpInfo = {
+  dbName: 'infinum_boilerplate_dev',
+  dbUser: 'wp',
+  dbPass: "wp",
+  wpUser: "ivan.grginov",
+  wpPass: "wp-pass",
+  siteName: "ImeStranice"
```

Comment:
This is a test or? If any default text will be placed it should be in English :)

Reply:
Temp stuff, there's really no need to set any defaults here, will remove. 👍